packages/xsnap/src/avaAssertXS.js

```diff
@@ -1,4 +1,6 @@
 /* global globalThis */
+/* We make exceptions for test code. This is a test utility. */
+/* eslint-disable @jessie.js/no-nested-await */
 /* eslint-disable no-await-in-loop */
```

Comment:
Consider putting on one line:
```suggestion
/* eslint-disable no-await-in-loop, @jessie.js/no-nested-await -- test code */
```